backend/controllers/WebhookController.ts

```diff
@@ -72,9 +74,13 @@ export class WebhookController extends BaseController {
     }
   }
 
-  public postAnnouncementsWebhook(req: Request, res: Response) {
+  public async postAnnouncementsWebhook(req: Request, res: Response) {
     ContentRepository.addAnnouncement(req.body);
 
+    logger.warn({ announcement: req.body }, 'postAnnouncementsWebhook: Announcement received');
+    logger.warn('postAnnouncementsWebhook: sseManager Broadcasting announcement');
```

Comment:
We can remove or make these more appropriate log levels.
```suggestion
    logger.debug({ announcement: req.body }, 'postAnnouncementsWebhook: Announcement received');
    logger.trace('postAnnouncementsWebhook: sseManager Broadcasting announcement');
```